tables/he.utb

```diff
@@ -0,0 +1,420 @@
+# liblouis: Hebrew table which supports multi-language texts and it is used in Israel.
+#
+# Copyright (C) 2020, BAUM Engineering <liblouis@baum.ro>
+#
+# This file is part of liblouis.
+#
+# liblouis is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 2.1 of the
+# License, or (at your option) any later version.
+#
+# liblouis is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with liblouis. If not, see
+# <http://www.gnu.org/licenses/>.
+
+#-index-name: Hebrew, uncontracted
+#-display-name: Hebrew braille
+
+#+locale: he
+#+type: literary
+#+dots: 6
+#+contraction: no
+#+grade: 1
+
+#-license: LGPLv2.1
+#-author-name: BAUM Engineering
+#-author-email: liblouis@baum.ro
+
+include en-chardefs.cti All character definition opcodes
+
+# Braille indicators
+numsign 3456  number sign, just a dots operand
+
+capsletter 6
+begcapsword 6-6
+endcapsword 6-3
+
+# literary forms of the decimal digits
+include litdigits6Dots.uti
+
+# display/passthrough for unicode braille
+include braille-patterns.cti
+
+# Letters are defined in chardefs.cti
+
+# punctuation
+prepunc ( 2356
+word ( 2356
+always ( 2356
+postpunc ) 2356
+word ) 2356
+always ) 2356
+prepunc " 236
+postpunc " 356
+word " 236
+prepunc ` 6-236
+prepunc ' 6-236
+postpunc ' 356-3
+word ' 4-3
+word 'em = 
+word 'tis =
+word 'twas =
+begnum # 456-1456 print number sign before number
+midnum , 2
+postpunc , 2
+word , 2
+decpoint . 46
+always - 36
+postpunc . 256
+word . 256
+postpunc ; 23
+word ; 23
+midnum : 25
+postpunc : 25
+always : 25
+postpunc ! 235
+always ! 235
+midnum / 34
+always / 34
+always < 25-246
+always = 123456
+always > 135-25
+always + 346
+postpunc ? 26
+endword s' 234-3
+word ? 26
+endnum st 34
+endnum nd 1345-145
+endnum rd 1235-145
+noback endnum 's 3-234
+endnum th 1456
+endnum 's 3-234
+always % 25-1234
+midnum ^ 45
+always ^ 45
+always ~ 45
+always & 12346
+midnum * 4-16
+always * 35-35
+prepunc [ 6-2356
+word [ 6-2356
+always [ 6-2356
+postpunc ] 2356-3
+word ] 2356-3
+always ] 2356-3
+prepunc { 12346
+word { 12346
+always { 12346
+postpunc } 13456
+word } 13456
+always } 13456
+always @ 4
+always \\ 1256
+always | 1256
+always ..." 3-3-3-356
+always ... 3-3-3
+always ...' 3-3-3-356-3
+repeated .... 0
+repeated ____ 456
+repeated ---- 36
+always .\s.\s." 3-3-3-356
+always .\s.\s. 3-3-3 . . .
+begnum $ 1246
+nofor begnum $ 1246
+always $ 1246
```

Comment:
This makes no sense. I mean I understand where it comes from. But these three rules all have the same character to dot mapping so they could be replaced with only the `always` rule. The same remark can be made about some other cases above.